### ISSUE TITLE
Class startup experience

### ISSUE THREAD

**Issue by betatim** [earthlab/hub-ops#107]
We discussed that students would have to wait while the cluster scales up and how to solve this by logging in before class etc.

A much simpler solution is to visit the [admin panel](https://hub.earthdatascience.org/bootcamp-hub/hub/admin) and start the servers for students there. Requires a few clicks and will cause the cluster to scale up before class. We can add users before they login for the first time if we know their usernames.

cc @lwasser

**Comment by lwasser** [comment]
Oh! That sounds perfect . How do I do this? I tried to add admins Friday and had issues
On Aug 5, 2018, at 1:39 AM, Tim Head <notifications@github.com<mailto:notifications@github.com>> wrote:

We discussed that students would have to wait while the cluster scales up and how to solve this by logging in before class etc.

A much simpler solution is to visit the admin panel<https://hub.earthdatascience.org/bootcamp-hub/hub/admin> and start the servers for students there. Requires a few clicks and will cause the cluster to scale up before class. We can add users before they login for the first time if we know their usernames.

cc @lwasser<https://github.com/lwasser>
